addons/crm/models/crm_lead.py

```diff
@@ -403,7 +403,7 @@ def create(self, vals):
 
         result = super(Lead, self.with_context(context)).create(vals)
         # Compute new probability for each lead separately
-        result._write_probability(vals)
+        result._write_probability(vals, force_compute=True)
```

Comment:
Shouldn't we avoid a force compute brute force and instead build a new pls_values dictionary, taking the lead values for each key present in ['tag_ids', 'stage_id', 'team_id'] + self._pls_get_safe_fields() ?

cc @dbeguin

Reply:
The issue here is that, when creating via the kanban view, the stage is not set in vals (the DB will set the default stage_id value when creating the record). The idea was that at create, we know that we should recompute anyway. But while I'm writing this,.. result should have all the values. We could add at least stage_id in the vals. But I think this could be overkill to check all the values for each key present in ['tag_ids', 'stage_id', 'team_id'] + self._pls_get_safe_fields().. and adding only stage_id to force recompute is quite weird and arbitrary.

This is why I think force_recompute may be the simplest and quickest solution (and it's quite self speaking)